erpnext/accounts/doctype/payment_entry/payment_entry.py

```diff
@@ -265,6 +266,25 @@ def validate_reference_documents(self):
 						frappe.throw(_("{0} {1} must be submitted")
 							.format(d.reference_doctype, d.reference_name))
 
+	def validate_paid_invoices(self):
+		no_oustanding_refs = {}
+
+		for d in self.get("references"):
+			if not d.allocated_amount:
+				continue
+
+			if d.reference_doctype in ("Sales Invoice", "Purchase Invoice", "Fees"):
+				outstanding_amount = frappe.get_cached_value(d.reference_doctype, d.reference_name, "outstanding_amount")
+				if outstanding_amount <= 0:
```

Comment:
If credit note has issued then invoice has the negative outstanding amount 

<img width="1208" alt="Screenshot 2020-04-29 at 9 41 32 PM" src="https://user-images.githubusercontent.com/8780500/80619413-4e1a5c80-8a62-11ea-8508-b57f0914b976.png">

Need to handle the case of credit / debit note